ert_shared/storage/storage_api.py

```diff
@@ -206,6 +208,10 @@ def set_observation_attribute(self, name, attribute, value):
     def get_ensemble(self, ensemble_id):
         with self._rdb_api as rdb_api:
             ens = rdb_api.get_ensemble_by_id(ensemble_id)
+            
+            if ens == None:
```

Comment:
my opinion - i think it looks best like this
```
if ensemble is None:
    return None
```

in python you can do both `is` and `==`  so you are free to choose, but other code in this file already does `is` and it is nice if all devs is adhering to the same code styles ( all devs usually have different opinions about code style, so there is not one "correct" answer )

Reply:
It's not actually just a style preference, one should always use `is None`  or `is not None` when comparing to `None`:slightly_smiling_face: For the interested reader: http://jaredgrubb.blogspot.com/2009/04/python-is-none-vs-none.html